lets-revise-hooks-today.md

```diff
@@ -0,0 +1,656 @@
+Hello Everyone
+
+I'm am Minhaj Ahmad Khan and I'm a Front-End developer. After a good response to my first blog. Thanks for your like and support so here I am back with a new blog I am sure you like it.
+
+## Why Hooks?
+
+Until now if you started writing a functional component and ran into a situation where you needed to add a state you would have to convert the component to a class component that is simply because the state could only be used in class components That where Hooks comes into the picture.
+
+## What is Hooks?
+
+A Hook is a special function that lets you “hook into” React features. For example, useState is a Hook that lets you add React state to function components.
```

Comment:
keep it simple. ... let's you use class bases functionalities into functional component